Formula/kyma-cli.rb

```diff
@@ -0,0 +1,38 @@
+class KymaCli < Formula
+  desc "Kyma command-line interface"
+  homepage "https://kyma-project.io"
+  url "https://github.com/kyma-project/cli.git",
+      :tag      => "v0.7.0",
+      :revision => "07d1041389d1cb0b8a9e69cda40e3afa762bbc72"
+  head "https://github.com/kyma-project/cli.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/kyma-project/cli/"
+    bin_path.install Dir["*"]
+
+    cd bin_path do
+      system "make", "resolve"
+      system "make", "build"
+      bin.install "bin/kyma-darwin"
+      mv bin/"kyma-darwin", bin/"kyma"
+    end
+  end
+
+  def caveats
+    <<~EOS
+      Happy Kyma-ing! Type \033[1;31mkyma help\033[0m to start
```

Comment:
This isn't homebrew specific. Please remove

Reply:
Removed